### ISSUE TITLE
Loading over the network

### ISSUE THREAD

**Issue by gedw99** [hemanthrajv/flutter_3d_obj#4]
Is it possible to load the .object over the network instead of from an embedded asset ?

**Comment by hemanthrajv** [comment]
Yes, for now you can use dart:http package to download the .obj to temp
folder and pass the path. Will try to get the package accept then asap and
PRs are welcome.

**Comment by gedw99** [comment]
hey @hemanthrajv  
I think the flutter cache plugin should then be used with it. That downloads the file from the server and puts it in temp dir for the app and will remove it automatically based on the headers from the server.

example usage of the cache:
https://github.com/renefloor/flutter_cached_network_image

The actual cache:
https://github.com/renefloor/flutter_cache_manager

Hoping you think this is a good idea... ?
